azurerm/resource_arm_managed_disk_test.go

```diff
@@ -131,6 +131,17 @@ func TestAccAzureRMManagedDisk_update(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testCheckAzureRMManagedDiskDestroy,
 		Steps: []resource.TestStep{
+			{
```

Comment:
@katbyte thanks for your reply. I'll take a look at it.

Reply:
@katbyte Is it possible to run acceptance test   authenticating using the Azure CLІ as I have no ability to create service principal?

Reply:
@tombuildsstuff I'm not able to run acceptance tests as I can login only with az cli and have no permissions to create service principal. So have no ability to write acceptance tests.   I can delete the changes I added to resource_arm_managed_disk_test.go or if you don't mind I can spent some time to trying to run it without sp(but I don't know how much it will take). So what would you recommend to do?

Reply:
@ira-bryndzei it's probably easiest to revert the additional test block in this file (but leave the updated constant/enum name) for the moment (we can then push a new test for this new disk type in a follow up commit) - what do you think?

Reply:
@tombuildsstuff that's work for me. I'll revert that block

Reply:
@tombuildsstuff I deleted that block

Reply:
👍 thanks - I've pushed a couple of commits (one to add the new values to the docs, which I'd missed before; and another to add the tests as discussed above) - I hope you don't mind!

Reply:
great! thanks